app/src/main/java/com/hedvig/app/feature/referrals/ui/tab/ReferralsAdapter.kt

```diff
@@ -462,14 +471,35 @@ class ReferralsAdapter(
                 data: ReferralsModel,
                 reload: () -> Unit,
                 tracker: ReferralsTracker,
-                marketManager: MarketManager
+                marketManager: MarketManager,
+                onBannerClicked: () -> Unit
```

Comment:
This could perhaps be added directly when creating the `ViewHolder` instead of upon each call to `bind()`.